Parser/pegen/pegen.c

```diff
@@ -937,7 +937,7 @@ _PyPegen_number_token(Parser *p)
 static int // bool
 newline_in_string(Parser *p, const char *cur)
 {
-    for (char c = *cur; cur >= p->tok->buf; c = *--cur) {
+    for (char c = *cur; cur > p->tok->buf; c = *--cur) {
         if (c == '\'' || c == '"') {
             return 1;
         }
```

Comment:
Would this make the code a bit easier to understand? I don't exactly have a hard time understanding it but I certainly need a second glance over it now.

```suggestion
    for (const char *c = cur; c >= p->tok->buf; c--) {
        if (*c == '\'' || *c == '"') {
            return 1;
        }
```

Reply:
Yeah, I like this version more :+1: